inference-engine/ie_bridges/python/install/setup.py.in

```diff
@@ -0,0 +1,249 @@
+                    "--component", python_version])
+
+        self.announce("Installing ngraph python api", level=3)
+        self.spawn(["cmake",
+                    "--install", CMAKE_BUILD_DIR,
+                    "--prefix", CMAKE_INSTALL_BIND_DIR,
+                    "--component", f"pyngraph_{python_version}"])
+
+        self.announce("Installing tbb", level=3)
+        self.spawn(["cmake",
+                    "--install", CMAKE_BUILD_DIR,
+                    "--prefix", CMAKE_INSTALL_TBB_DIR,
+                    "--component", "tbb"])
+        self.announce("Configuring libraries", level=3)
+
+        self.announce("Configuring libraries", level=3)
+        if sys.platform != "win32":
+            rpath = ""
+            lib_pattern = ""
+            if sys.platform == "linux":
+                rpath = "$ORIGIN"
+                lib_pattern = "*.so"
+            elif sys.platform == "darwin":
+                rpath = "@loader_path"
+                lib_pattern = "*.dylib"
+            for lib in RELOCATABLE_LIBS:
+                for path in Path(lib).glob(lib_pattern):
+                    set_rpath(rpath, path)
+
+
+class CopyExt(build_ext):
+    def run(self):
+        for extension in self.extensions:
+            if not isinstance(extension, PrebuiltExtension):
+                raise DistutilsSetupError(
+                    "copy_ext can accept PrebuiltExtension only")
+            src = extension.sources[0]
+            dst = self.get_ext_fullpath(extension.name)
+            try:
+                os.makedirs(os.path.dirname(dst), exist_ok=True)
+            except OSError as exc:
+                if exc.errno != errno.EEXIST:
+                    raise
+
+            #setting relative path to find dlls
+            if sys.platform != "win32":
+                rpath = os.path.relpath(BINDINGS_DIR, os.path.dirname(src))
+                if sys.platform == "linux":
+                    rpath = os.path.join("$ORIGIN", rpath, WHEEL_LIBS_INSTALL_DIR)
+                elif sys.platform == "darwin":
+                    rpath = os.path.join("@loader_path", rpath, WHEEL_LIBS_INSTALL_DIR)
+                set_rpath(rpath, src)
+
+            copy_file(src, dst, verbose=self.verbose, dry_run=self.dry_run)
+
+def set_rpath(rpath, executable):
+    print(f"Setting rpath {rpath} for {executable}")
+    cmd = ""
+    if sys.platform == "linux":
+        cmd = ["patchelf", "--set-rpath", rpath,  executable]
+    elif sys.platform == "darwin":
+        cmd = ["install_name_tool", "-add_rpath", rpath,  executable]
+    ret_info = subprocess.run(cmd)
+    if ret_info.returncode != 0:
+        print(f"The exit code: {ret_info.returncode}")
+        print(f"Could not set rpath: {rpath} for {executable}")
+        exit(2)
+
+
+def find_data_files(src_dirs, install_subdir):
+    data_files = []
+    # excluding unneeded libs for wheel distribution
+    data_blacklist = ['.lib', '.pdb', '_debug.dll']
```

Comment:
probably only required files should be found

Reply:
This should be fixed in "cmake install"(@slyubimt fyi), here we just implemented a workaround to filter unneeded content

Reply:
Please add a comment in this case

Reply:
fixed